client/views/list/list.js

```diff
@@ -693,5 +693,8 @@ Template.list.events({
   'blur .replyarea': function(event, template) {
     Template.instance().state.set('typing', false);
   },
+  'click .closecontainer': function(event, template) {
+    $('.admincontainer').css('display', 'none'); // Updated the css on the click event of close button in admin interface card
```

Comment:
@singhsanket143 can you remove the comment? It's totally self-explanatory.

Reply:
@peter-hank I have removed the comment. Please review it.